Jenkinsfile

```diff
@@ -79,6 +79,99 @@ pipeline {
+               defaultValue: '',
+               description: 'Package version to use instead of building. example: 1.3.103-1, 1.2-2')
+        string(name: 'CI_HARDWARE_DISTRO',
+               defaultValue: 'centos7',
+               description: 'Distribution to use for CI Hardware Tests')
+        string(name: 'CI_CENTOS7_TARGET',
+               defaultValue: 'el7',
+               description: 'Image to used for Centos 7 CI tests')
+        string(name: 'CI_CENTOS8_TARGET',
+               defaultValue: 'el8.3',
+               description: 'Image to used for Centos 8 CI tests')
+        string(name: 'CI_LEAP15_TARGET',
+               defaultValue: 'leap15.2',
+               description: 'Image to use for OpenSUSE Leap CI tests')
+        string(name: 'CI_UBUNTU20.04_TARGET',
+               defaultValue: 'ubuntu20.04',
+               description: 'Image to used for Ubuntu 20 CI tests')
+        booleanParam(name: 'CI_RPM_el7_NOBUILD',
+                     defaultValue: false,
+                     description: 'Do not build RPM packages for CentOS 7')
+        booleanParam(name: 'CI_RPM_el8_NOBUILD',
+                     defaultValue: false,
+                     description: 'Do not build RPM packages for CentOS 8')
+        booleanParam(name: 'CI_RPM_leap15_NOBUILD',
+                     defaultValue: false,
+                     description: 'Do not build RPM packages for Leap 15')
+        booleanParam(name: 'CI_DEB_Ubuntu20_BUILD',
+                     defaultValue: false,
+                     description: 'Do not build DEB packages for Ubuntu 20')
+        booleanParam(name: 'CI_ALLOW_UNSTABLE_TEST',
+                     defaultValue: false,
+                     description: 'Continue testing if a previous stage is Unstable')
+        booleanParam(name: 'CI_UNIT_TEST',
+                     defaultValue: true,
+                     description: 'Run the Unit CI tests')
+        booleanParam(name: 'CI_UNIT_TEST_MEMCHECK',
+                     defaultValue: true,
+                     description: 'Run the Unit Memcheck CI tests')
+        booleanParam(name: 'CI_FUNCTIONAL_el7_TEST',
+                     defaultValue: true,
+                     description: 'Run the functional CentOS 7 CI tests')
+        booleanParam(name: 'CI_MORE_FUNCTIONAL_PR_TESTS',
+                     defaultValue: false,
+                     description: 'Enable more distros for functional CI tests')
+        booleanParam(name: 'CI_FUNCTIONAL_el8_TEST',
+                     defaultValue: true,
+                     description: 'Run the functional CentOS 8 CI tests')
+        booleanParam(name: 'CI_FUNCTIONAL_leap15_TEST',
+                     defaultValue: true,
+                     description: 'Run the functional OpenSUSE Leap 15 CI tests')
+        booleanParam(name: 'CI_FUNCTIONAL_ubuntu20_TEST',
+                     defaultValue: false,
+                     description: 'Run the functional Ubuntu 20 CI tests')
```

Comment:
Do we want to explicitly label these as VM tests so they aren't confused with the HW distro?